rasa_sdk/interfaces.py

```diff
@@ -386,7 +386,9 @@ class ActionMissingDomainException(Exception):
 
     def __init__(self, action_name: Text, message: Optional[Text] = None) -> None:
         self.action_name = action_name
-        self.message = message or "Domain context is missing."
+        self.message = (
+            message or "Missing domain context, assistant will retry the request."
```

Comment:
Should we add a brief user-facing explanation why domain might be missing, maybe even a link to the docs where this would be explained?
```suggestion
            message or "Missing domain context, assistant will retry the request and include the domain into the request payload."
```

Reply:
I wanted to do it like this initially, but thought I should make it more concise and shorter. I'm up for having a more detailed message. I'll edit it.

Regarding the link to the docs, you mean this PR should wait for the docs to be ready so it could be linked here?

Reply:
> Regarding the link to the docs, you mean this PR should wait for the docs to be ready so it could be linked here?

You can already add a link if you know the section where the docs will go.

Reply:
I assume it will be included [here](https://rasa.com/docs/rasa-pro/action-server/) because we're describing the request like this:

> When your assistant predicts a custom action, the Rasa server sends a POST request to the action server with a json payload including the name of the predicted action, the conversation ID, the contents of the tracker and the contents of the domain.

We will edit the last part of the sentence that mentions the domain being included in the payload.

Reply:
Sounds good, although I'd add an extensive paragraph to describe both how the assistant operates now and the motivation and performance advantages for this new approach.

Reply:
Okay, I'll go with this link, but if we decide we want to have a dedicated page just for this behavior that would be on another link, I'll make sure to update this.